README.md

```diff
@@ -588,14 +625,17 @@ To begin using the `TargetingFilter` in an application it must be added to the a
 
 The implementation type used for the `ITargetingContextAccessor` service must be implemented by the application that is using the targeting filter. Here is an example setting up feature management in a web application to use the `TargetingFilter` with an implementation of `ITargetingContextAccessor` called `HttpContextTargetingContextAccessor`.
 
-```
+``` C#
 services.AddFeatureManagement();
+        .AddFeatureFilter<BrowserFilter>()

```

Comment:
Why add the BrowserFilter in the targeting example?

Reply:
I want to make it more clear how to call the `WithTargeting<>()` method. We didn't mention `AddFeatureFilter` method will return `IFeatureManagementBuilder`.

Reply:
Removed.